### PR TITLE
Remove extension for Terms in Values in Subst

### PR DESCRIPTION
**Pull request by mariari**
This clears up the evaluator substv code by removing an unused constraint. Namely the `extT` for term extensions. This extension was placed due to the `Error` type being too broad. For clarity and debugging reasons this was removed.

The main drawback besdies readability is that this under-constrained the `extT` type which made it imposible to print any terms handed to the code.

### CODE REVIEW

File: library/EasyPipeline/src/Easy.hs [hunk 1]
```diff
@@ -564,9 +564,11 @@ printTimeLapse byteString option = do
   --
   (_, cored) <- coreify byteString option
   traverse_ (printCoreFunction cored option) currentDefinedItems
+  print "finished Cored"
```
Comment (cwgoes) [inline library/EasyPipeline/src/Easy.hs:567]: remove this before merge? or do you want to keep the debugging info in the easy pipeline?

Comment (mariari) [reply library/EasyPipeline/src/Easy.hs:567]: This is useful to have for the timelapse as it came under question what a pass was.

**Comment by cwgoes** [comment]
Code looks fine but can you reference an issue or briefly explain in the top comment the reason for this change?

**Comment by mariari** [comment]
> Code looks fine but can you reference an issue or briefly explain in the top comment the reason for this change?

Updated

**Review by cwgoes — APPROVED** [review]
utACK

**Pull request merged**